lazy-pages/src/lib.rs

```diff
@@ -111,25 +157,88 @@ pub fn get_lazy_pages_numbers() -> Vec<PageNumber> {
     LAZY_PAGES_CONTEXT.with(|ctx| ctx.borrow().lazy_pages_info.keys().copied().collect())
 }
 
-/// Returns current wasm mem buffer pointer, if it's set
-pub fn get_wasm_mem_addr() -> Option<HostPointer> {
+#[derive(Debug, derive_more::Display)]
+pub enum InitForProgramError {
+    #[display(
+        fmt = "Wasm mem native addr {:#x} is not aligned to the native page size",
```

Comment:
```suggestion
        fmt = "WASM memory native address {:#x} is not aligned to the native page size",
```
Same for variants below